low-level/files/opt/seagate/sspl/conf/sspl.init.tmpl.1-node

```diff
@@ -0,0 +1,57 @@
+cluster:
+  cid01:
```

Comment:
This is cluster-id right, and not sample file. some TMPL_CLUSTER_ID_01 ?

Reply:
Done.